mmseg/core/evaluation/metrics.py

```diff
@@ -19,6 +21,21 @@ def intersect_and_union(pred_label, label, num_classes, ignore_index):
          ndarray: The ground truth histogram on all classes.
     """
 
+    if isinstance(pred_label, str):
+        pred_label = np.load(pred_label)
+
+    if isinstance(label, str):
+        label = mmcv.imread(label, flag='unchanged', backend='pillow')
+        # modify if custom classes
+        if label_map is not None:
+            for old_id, new_id in label_map.items():
+                label[label == old_id] = new_id
+        if reduce_zero_label:
+            # avoid using underflow conversion
+            label[label == 0] = 255
+            label = label - 1
+            label[label == 254] = 255
```

Comment:
Could `reduce_zero_labels` be merged with `label_map`?

Reply:
But in [custom.py](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/custom.py#L95-L96), ```reduce_zero_label``` and ```label_map``` is two different attributes. If we merge the two different attributes in [metrics.py](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/core/evaluation/metrics.py) but retain the two different attributes in [custom.py](https://github.com/open-mmlab/mmsegmentation/blob/master/mmseg/datasets/custom.py#L95-L96), is it something disunity?